java.js

```diff
@@ -183,4 +185,25 @@ function checkLowerCase(input) {
 form.addEventListener('submit', (event) => {
   event.preventDefault();
   checkLowerCase(email);
-});
\ No newline at end of file
+});
+
+const valueInput = JSON.parse(localStorage.getItem('valueInput'));
+
+function addVale(n, e, t) {
+  valueInput.push({ n, e, t });
+  localStorage.setItem('valueInput', JSON.stringify(valueInput));
+  return { n, e, t };
+}
+
+function showNow({ n, e, t }) {
+  name.value = n;
+  email.value = e;
+  text.value = t;
+}
+
+valueInput.forEach(showNow);
+
+form.addEventListener('change', (event) => {
+  event.preventDefault();
+  addVale(name.value, email.value, text.value);
+});
```

Comment:
- [ ] Your form is not being persistent to refresh, kindly check the video below :arrow_double_down: 
![image](https://user-images.githubusercontent.com/62705947/209353948-c723e5a8-5ef7-41d2-b623-64ac0c7fc2d9.png)

https://www.loom.com/share/afb3a3e3670f4ee2a28eb56c61be8f8e